### ISSUE TITLE
Validation data explanation / strategy?

### ISSUE THREAD

**Issue by RobertSellers** [CIA-Oceanix/GeoTrackNet#3]
Hi again,

I'm curious as to the data you've entered into your model and what your validation dataset's purpose is.  Are you providing examples of "valid" (normal)  tracks or "invalid"(anomalous) tracks exclusively? Is there a strategy that you've done, or would my suggested datasets make sense as a validation dataset?

**Comment by duonginspace** [comment]
Hi,
The validation set serves 2 purposes: 
1. To prevent overfitting (by comparing the average log likelihood on the training set and on the validation set)
2. To calculate the distribution in each small cell, which will be used in the local thresholding step of the _a contrario_ detection. Please see Section III-C of this paper https://arxiv.org/pdf/1912.00682.pdf for details.

**Comment by RobertSellers** [comment]
I appreciate the feedback. Thank you.

**Comment by muzunpostalci** [comment]
Hello @dnguyengithub ,

Can you further elaborate on why the validation set is used to calculate the distribution? I observed that the _a contrario_ detection is dependent on the distribution of each cell but I didn't get why the validation set is a good candidate for it. Does using training set for this matter lead to overfitting?
